src/Node/Jwt.purs

```diff
@@ -103,7 +101,13 @@ foreign import _decode :: Fn3 (Foreign -> Maybe Foreign) (Maybe Foreign) String
 decode :: forall r l. Decodable r l => String -> Either (NonEmptyList String) (Token r Unverified)
 decode s = (note (singleton "Couldn't decode token") $ runFn3 _decode Just Nothing s) >>= foreignToToken
 
+decode' :: String -> Either (NonEmptyList String) (Token () Unverified)
+decode' = decode
+
 foreign import _verify :: Fn4 (Foreign -> Maybe Foreign) (Maybe Foreign) String String (Maybe Foreign)
 
 verify :: forall r l. Decodable r l => Secret -> String -> Either (NonEmptyList String) (Token r Verified)
-verify (Secret secret) s = (note (singleton "Couldn't verify token") $ runFn4 _verify Just Nothing secret s) >>= foreignToToken
+verify (Secret secret) s = (note (singleton "Couldn't verify token") $ runFn4 _verify Just Nothing s secret) >>= foreignToToken
+
+verify' :: Secret -> String -> Either (NonEmptyList String) (Token () Unverified)
+verify' = verify
```

Comment:
decode' and verify' are useful when dealing with token that have no unregistered claims, or when the unregistered claims don't matter.